base/io.jl

```diff
@@ -645,3 +645,34 @@ ismarked(io::IO) = io.mark >= 0
 Commit all currently buffered writes to the given stream.
 """
 flush(io::IO) = nothing
+
+"""
+    skipchars(io::IO, predicate; linecomment=nothing)
```

Comment:
Oh my bad. I didn't see that either. How should I proceed here? Restore branch and remove one of the docstrings?